cli/cmd/cloud_account.go

```diff
@@ -182,18 +184,73 @@ func buildDetailsTable(integration api.V2RawType) string {
 				for i, j := range val {
 					if v, ok := j.(string); ok {
 						details = append(details, []string{strings.ToUpper(format.SpaceUpperCase(i)), v})
+					} else {
+						cli.Log.Warn("unable to build table details, unknown type", "type", i, "key", k)
 					}
 				}
 			case []any:
 				var values []string
 				for _, i := range val {
-					values = append(values, i.(string))
+					if _, ok := i.(string); ok {
+						values = append(values, i.(string))
+					} else if _, ok := i.(map[string]interface{}); ok {
+						for m, n := range i.(map[string]interface{}) {
+							values = append(values, fmt.Sprintf("%s:%s", m, n))
+						}
+					} else {
+						cli.Log.Warn("unable to build table details, unknown type", "type", i, "key", k)
+					}
 				}
 				details = append(details, []string{strings.ToUpper(format.SpaceUpperCase(k)), strings.Join(values, ",")})
 			}
 		}
 	}
 
+	if caMap, ok := integration.GetProps().(map[string]interface{}); ok {
```

Comment:
This look good. Thanks for adding the props field.

I think the only missing part is the Server Token that is only available to inline scanner and proxy scanners, so it should not be added to the `V2RawType` struct but instead, identify if the provider interface `integration` is of type `ContainerRegistryRaw` and if so, cast it and access the server token information.

Reply:
I added a handler for the `ContainerRegistryRaw` type and output the `SERVER_TOKEN` and `SERVER_TOKE_URI`:
https://github.com/lacework/go-sdk/blob/75c5c324238291cbefe1694de75eb61cbbbb294e/cli/cmd/cloud_account.go#L254-L258

Do we need the URI, I don't know but might as well expose it here while we're at it.